assets/js/google-tag.js

```diff
@@ -0,0 +1,15 @@
+---
+layout: blank
+---
```

Comment:
why do we need a layout here

Reply:
This is actually a workaround for two things:

- We can't have the snippet in this file inside `main_scripts.html` because of our csp blocking inline scripts
- We want to somehow be able to reference the `google_analytics_ga4` value, which is in the jekyll specific site config

The solution to this is import this file as a script, but also provide this snippet with jekyll front matter specifying a blank layout, which allows us to reference `site.google_analytics_ga4`

Reply:
perhaps we should jot down this explanation in the file? it seems important enough, especially as we are looking to migrate off jekyll in the future

Reply:
yeah that makes sense, added in [a8348e6](https://github.com/isomerpages/isomerpages-template/pull/294/commits/a8348e68a244dca754c6d56fc42cdf803b082eeb)